packages/legend-extension-dsl-persistence/src/graph/metamodel/pure/model/packageableElements/persistence/DSL_Persistence_Partitioning.ts

```diff
@@ -36,8 +37,12 @@ export class NoPartitioning extends Partitioning {
 export abstract class FieldBased extends Partitioning {}
 
 export class FieldBasedForGraphFetch extends FieldBased {
+  partitionFieldPaths!: object[];
```

Comment:
same comment, please annotate this discrepancy

Reply:
Updated.